### PR TITLE
Documentation: Clarify basic regexp syntax of TERM in todo.sh list

### PR DESCRIPTION
**Pull request by inkarkat**
And give hints about escaping.
Issue raised by https://github.com/todotxt/todo.txt-cli/discussions/334

- [x] Fork [the repository](https://github.com/todotxt/todo.txt-cli) and create your branch from `master`.
- [ ] If you've added code that should be tested, add tests!
- [ ] Ensure the test suite passes.
- [ ] Format your code with [ShellCheck](https://www.shellcheck.net/).
- [x] Include a human-readable description of what the pull request is trying to accomplish.
- [x] Steps for the reviewer(s) on how they can manually QA the changes.
- [ ] Have a `fixes #XX` reference to the issue that this pull request fixes.

### CODE REVIEW

File: todo.sh [hunk 1]
```diff
@@ -217,9 +217,13 @@ actionsHelp()
 		      Displays all tasks that contain TERM(s) sorted by priority with line
 		      numbers.  Each task must match all TERM(s) (logical AND); to display
 		      tasks that contain any TERM (logical OR), use
-		      "TERM1\|TERM2\|..." (with quotes), or TERM1\\\|TERM2 (unquoted).
+		      'TERM1\|TERM2\|...' (with quotes), or TERM1\\\|TERM2 (unquoted).
```
Comment (karbassi) [inline todo.sh:220]: Why the quote change? Is it important?

Comment (inkarkat) [reply todo.sh:220]: Not directly; for the following pipe character, it doesn't matter. But for other characters (dollar, backslash, backtick, ...) it does matter.
Also, I imagine this information is most important to beginners, who maybe use the Linux command-line just for todo.sh and not much else (yet). As single quoting has definite benefits (no doubling of backslashes) for the following (added) basic regexp escaping, I think it's best to just stick to single-quoting consistently here, or the alternative no-quoting, but not introduce a third alternative with double-quoting. That would just be too much for the beginner, who's just looking for an easy answer to get his searches working.

**Review by karbassi — APPROVED** [review]

**Comment by inkarkat** [comment]
@karbassi It seems like I cannot merge without the _travis-ci_ check passing, not here in GitHub, nor from the command-line:

> remote: error: GH006: Protected branch update failed for refs/heads/master.
> remote: error: Required status check "continuous-integration/travis-ci" is expected.

If I remember correctly, that Travis check has been replaced by the two GitHub actions, anyway. Can you please remove it?

**Comment by karbassi** [comment]
I wish I could. It's been a bug for over a year. I have no idea how to remove it.

**Comment by inkarkat** [comment]
Oh, gosh, well, so you're force-pushing the merged branch, then?

**Pull request merged**